### PR TITLE
[DATAPLT-2456] Fork Polynote with Necessary Changes

### PR DESCRIPTION
**Pull request by eschranz**
In order to use Spark 3.3.1, we need to fork from the main Polynote repo and make some changes. This PR tracks those changes.

### CODE REVIEW

File: docker/dev/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,14 @@
 # Set this to match your JDK version compiled with.
-FROM openjdk:8-slim-buster
+FROM --platform=linux/x86_64 openjdk:11-slim-buster
```
Comment (eschranz) [inline docker/dev/Dockerfile:2]: `--platform=linux/x86_64` should be removed once we have Butler responsible for the build.

File: docker/dev/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 
 WORKDIR /opt
 
 RUN apt update -y && \
-    apt install -y wget python3 python3-dev python3-pip build-essential
+    apt install -y wget python3 python3-dev python3-pip build-essential git curl nano vim sudo
```
Comment (eschranz) [inline docker/dev/Dockerfile:7]: I added `sudo` on top of the other additions made previously.

File: docker/dev/Dockerfile [hunk 3]
```diff
@@ -23,9 +22,9 @@ RUN pip3 install -r /tmp/requirements.txt
 # Package everything together by making the distribution with `sbt dist`
 # Then build this from the project root using `-f docker/dev/Dockerfile` to select this file.
 # for example (don't forget the dot at the end!):
-#   cd polynote-static; npm install; npm run dist; cd..
+#   cd polynote-frontend; npm install; npm run dist; cd..
```
Comment (alexcsolomon) [inline docker/dev/Dockerfile:25]: I dont *think* we need to do this part

File: docker/dev/Dockerfile [hunk 4]
```diff
@@ -23,9 +22,11 @@ RUN pip3 install -r /tmp/requirements.txt
 # Package everything together by making the distribution with `sbt dist`
 # Then build this from the project root using `-f docker/dev/Dockerfile` to select this file.
 # for example (don't forget the dot at the end!):
-#   cd polynote-static; npm install; npm run dist; cd..
+# for frontend dev
```
Comment (eschranz) [inline docker/dev/Dockerfile:25]: Added for clarification

File: docker/dev/Dockerfile (hunk 1 above)
Comment (alexcsolomon) [reply docker/dev/Dockerfile:2]: +1

File: docker/dev/Dockerfile (hunk 2 above)
Comment (alexcsolomon) [reply docker/dev/Dockerfile:7]: gracias

File: docker/dev/Dockerfile (hunk 4 above)
Comment (alexcsolomon) [reply docker/dev/Dockerfile:25]: thanks!

**Review by alexcsolomon — APPROVED** [review]

**Pull request merged**